actors/builtin/sector.go

```diff
@@ -17,44 +17,44 @@ const EpochsInFiveYears = stabi.ChainEpoch(5 * EpochsInYear)
 var SealProofPolicies = map[stabi.RegisteredSealProof]*SealProofPolicy{
 	stabi.RegisteredSealProof_StackedDrg2KiBV1: {
 		SectorMaxLifetime:      EpochsInFiveYears,
-		ConsensusMinerMinPower: stabi.NewStoragePower(0),
+		ConsensusMinerMinPower: stabi.NewStoragePower(10 << 40),
 	},
 	stabi.RegisteredSealProof_StackedDrg8MiBV1: {
 		SectorMaxLifetime:      EpochsInFiveYears,
-		ConsensusMinerMinPower: stabi.NewStoragePower(16 << 20),
+		ConsensusMinerMinPower: stabi.NewStoragePower(10 << 40),
```

Comment:
I thought lotus only overwrote 64 GiB sector minima to 10 TiB, does it do it for all proof types?

Reply:
duh just read the pr comment